unimol/unimol/utils/docking_utils.py

```diff
@@ -199,10 +234,28 @@ def ensemble_iterations(
         ]
 
 
-def rmsd_func(holo_coords, predict_coords):
+def rmsd_func(holo_coords: np.ndarray, predict_coords: np.ndarray, mol: Optional[Chem.Mol] = None) -> float:
+    """ Symmetric RMSD for molecules. """
     if predict_coords is not np.nan:
         sz = holo_coords.shape
-        rmsd = np.sqrt(np.sum((predict_coords - holo_coords) ** 2) / sz[-2])
+        if mol is not None:
+            # get stereochem-unaware permutations: (P, N)
+            base_perms = np.array(mol.GetSubstructMatches(mol, uniquify=False))
```

Comment:
symmetric rmsd based on potential rearrangements of the molecule that leave it the same graph